Comparison Engine/Comparison Engine/Form1.cs

```diff
@@ -53,127 +51,117 @@ private void Form1_Load(object sender, EventArgs e)
 
         //LIST FEATURES      #lists
 
-        private void initializeList()
+        private void InitializeList()
         {
-            stateCheck();
+            StateCheck();
         }   
```

Comment:
After all changes that we've done, it seems pointless to have a method just to call another method.

Reply:
done